client/l2tp_client.c

```diff
@@ -492,6 +499,19 @@ void context_process_control_packet(l2tp_context *ctx)
 
   // Check packet type
   switch (type) {
+    case CONTROL_TYPE_USAGE: {
+      if (ctx->state == STATE_GET_USAGE) {
+          // broker usage information
```

Comment:
This has some strange indentation? Also, comments should start with a capital letter.